src/EmployerAan/SFA.DAS.EmployerAan/Application/MemberProfiles/Queries/GetMemberProfileWithPreferences/GetMemberProfileWithPreferencesQueryHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MediatR;
+using SFA.DAS.EmployerAan.Common;
+using SFA.DAS.EmployerAan.Infrastructure;
+
+namespace SFA.DAS.EmployerAan.Application.MemberProfiles.Queries.GetMemberProfileWithPreferences;
+
+public class GetMemberProfileWithPreferencesQueryHandler : IRequestHandler<GetMemberProfileWithPreferencesQuery, GetMemberProfileWithPreferencesQueryResult>
+{
+    private readonly IAanHubRestApiClient _apiClient;
+
+    public GetMemberProfileWithPreferencesQueryHandler(IAanHubRestApiClient apiClient)
+    {
+        _apiClient = apiClient;
+    }
+
+    public async Task<GetMemberProfileWithPreferencesQueryResult> Handle(GetMemberProfileWithPreferencesQuery request, CancellationToken cancellationToken)
+    {
+        var responseMemberProfileWithPreferenceTask = _apiClient.GetMemberProfileWithPreferences(request.MemberId, request.RequestedByMemberId, request.IsPublicView, cancellationToken);
+        var responseMemberTask = _apiClient.GetMember(request.MemberId, cancellationToken);
+        var responseRegionsTask = _apiClient.GetRegions(cancellationToken);
+
+        await Task.WhenAll(responseMemberProfileWithPreferenceTask, responseMemberTask, responseRegionsTask);
+
+        GetMemberProfileWithPreferencesQueryResult result = new();
+
+        var outputMemberProfileWithPreferenceTask = responseMemberProfileWithPreferenceTask.Result;
```

Comment:
variable should not be suffixed with `Task` as it is the result of the API call. Just remove `Task` or I suggest `memberProfileWithPreferences` which can be suffixed with `Response` or `Result`